test-data/setup.sh

```diff
@@ -8,46 +8,70 @@
 
 
 # prerequisite: have keptn installed with --use-case=continuous-delivery flag
-
-# 1. install Istio https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-08/index.html?index=..%2F..index#2 
-# 2. install Keptn https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-08/index.html?index=..%2F..index#4
-# 3. configure Istio + Keptn  shttps://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-08/index.html?index=..%2F..index#5
-# 4. connect the Keptn CLI to the cluster https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-08/index.html?index=..%2F..index#6 
+# 1. install Keptn https://keptn.sh/docs/quickstart/#helm
```

Comment:
The old Quick Start guide is being taken down because it is hopelessly out of date so this xref is not going to work.  Quick Start is being replaced by a Killercoda exercise so you probably can't link to that.  The doc for installing Keptn using the Helm chart is https://keptn.sh/docs/install/helm-install/ but then we need to specify the access option and such.  Ugh!